powerauth-push-server/src/main/java/io/getlime/push/repository/PushDeviceRepository.java

```diff
@@ -59,19 +59,26 @@ public interface PushDeviceRepository extends CrudRepository<PushDeviceRegistrat
      * Find all device registrations by given user ID and app ID. This list represents all devices that a single user
      * has registered.
      * @param userId User ID.
-     * @param appId App ID.
+     * @param rid Credentials database record ID.
```

Comment:
Probably I am not yet familiar enough with PA and `rid` seems too short  and not self-explaining to me.